t/24-worker-jobs.t

```diff
@@ -1238,6 +1238,29 @@ subtest 'known images and files populated from status update' => sub {
     is_deeply($job->known_files,  \@fake_known_files,  'known files populated from status update');
 };
 
+subtest 'write no space left to job failed reason by parsing autoinst-log' => sub {
```

Comment:
```suggestion
subtest 'write no space left to job reason by parsing autoinst-log' => sub {
```

(Just because this is not really tied to the job result "failed".)